src/frontend/src/features/layouts/components/mailbox-panel/_index.scss

```
@@ -29,27 +29,45 @@
     }
 
     button.mailbox-panel__mailbox-title__dropdown-button {
-        color: var(--c--contextuals--content--semantic--neutral--primary);
-        font-weight: 700;
-        padding-block: var(--c--globals--spacings--sm);
+        padding-block: var(--c--globals--spacings--base) var(--c--globals--spacings--sm);
         // 1.1rem hardcoded to be aligned with other icon in the sidebar
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)